tbx/static_src/sass/components/_skip-link.scss

```diff
@@ -4,7 +4,7 @@
     position: absolute;
     top: -200px;
     left: 0;
-    color: $color--black;
+    color: $color--off-black;
```

Comment:
I imagine it will be. I was just updating references to variables that don't exist anymore (and have done in a few other places) expecting them all to be updated when we come to develop them.

Reply:
thanks, no worries